python_modules/dagster/dagster_tests/asset_defs_tests/test_freshness_policy.py

```diff
@@ -202,8 +202,17 @@ def foo():
     # Then, check that we can deserialize the old snapshot with new Dagster version
     old_snap_deserialized = deserialize_value(old_snap_serialized, RepositorySnap)
 
-    # Then, check that the deserialized snapshots are the same
-    assert new_snap == old_snap_deserialized
```

Comment:
agree with removing this check, seems like it is destined to break in the future on innocuous changes